L1Trigger/L1THGCal/src/backend/HGCalHistoClusteringImpl.cc

```diff
@@ -33,8 +33,7 @@ HGCalHistoClusteringImpl::HGCalHistoClusteringImpl(const edm::ParameterSet& conf
 
 float HGCalHistoClusteringImpl::dR(const l1t::HGCalCluster& clu, const GlobalPoint& seed) const {
   Basic3DVector<float> seed_3dv(seed);
-  GlobalPoint seed_proj(seed_3dv / seed.z());
-  return (seed_proj - clu.centreProj()).mag();
+  return (seed_3dv - clu.centreProj()).mag();
```

Comment:
Does not compile. Cannot make the difference of a point and a vector. The conversion to vector is not needed.

Reply:
Yes sorry, now fixed